proxy-explanation/draft.md

```diff
@@ -72,13 +73,14 @@ The "application/proxy-explanation+json" media type denotes a JSON {{RFC7159}} f
 * **name** - A short string identifying the party operating the proxy
 * **title** - A short string title for the explanation
 * **description** - A string explaining why the request wasn't successful
-* **moreinfo** - A string containing an absolute URL {{RFC3986}} which the user can find relevant information.
 
-The "name" and "title" members MUST be present; all other members are OPTIONAL.
+The "name" and "title" members MUST be present; all other members are OPTIONAL. All strings MUST be encoded in UTF-8 {{RFC3629}}.
+
+Proxies MUST NOT include any URL in any of the fields and client SHOULD prevent the display of obfuscated URLs to the end user.
```

Comment:
Hmm, I don't think either of these is enforceable; something more of a statement of intent (without RFC2119 requirements) would work, though.